library/btc24/btc24.ts

```diff
@@ -0,0 +1,88 @@
+import {
+  SimplePool,
+  finalizeEvent,
+  useWebSocketImplementation,
+} from "nostr-tools";
+import log from "loglevel";
+import db from "../db";
+const { encrypt } = require("nostr-tools/nip04");
+import { hexToBytes } from "@noble/hashes/utils";
+useWebSocketImplementation(require("ws"));
+
+interface ZapRequestEvent {
+  content: string;
+  tags: [string, string][];
+}
+
+const WAVLAKE_SECRET = hexToBytes(process.env.NOSTR_SECRET);
+const DM_RECEIVER_PUBKEY =
+  "5195cb1e807a4f412f8bef2a08258244f168c4903d1ab17675eb1f3a059f5921";
+const DM_RECEIVER_RELAYS = [
+  "wss://relay.primal.net",
+  "wss://relay.wavlake.com",
+];
+
+const getTrackMetadata = async (trackId: string) => {
+  const trackDetails = await db
+    .knex("track_info")
+    .select("title", "artist", "artwork_url")
+    .where("id", "=", trackId)
+    .then((data) => {
+      return data[0];
+    })
+    .catch((err) => {
+      log.error(`Error getting track metadata for ${trackId}: ${err}`);
+    });
+  return trackDetails;
+};
+
+export const handleConferenceZap = async (zapRequestEvent: ZapRequestEvent) => {
+  log.info("Handling conference zap request");
+  // Parse zap request for track id and message
+  const aTag = zapRequestEvent.tags.find((x) => x[0] === "a");
+  const comment = zapRequestEvent.content;
+  const amount = zapRequestEvent.tags.find((x) => x[0] === "amount");
+
+  const trackId = aTag[1].split(":")[2];
+
+  // Retrive track metadata (artwork, title, artist)
+  const trackDetails = await getTrackMetadata(trackId);
+
+  const messageObject = {
+    sats: amount,
```

Comment:
need to divide by 1000 to convert to sats
```suggestion
    sats: parseInt(amount) / 1000,
```